build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1407,6 +1412,11 @@ message ExecuteOperationMetadata {
   // being executed.
   Digest action_digest = 2;
 
+  // A unique identifier for an ongoing execution of the action, which will
+  // also be returned as `ActionResult.execution_metadata.execution_uuid`.
```

Comment:
Does the execution_metadata.execution_uuid have to match this one? Can the identifier change, either during the lifetime of the ExecuteOperationMetadata (e.g., if the server merges this action with another one) or at the end of execution?

Reply:
Yes, the identifier can change, e.g. through merges or restarts of the action. It should therefore match `execution_metadata.execution_uuid`.